Documentation/ApiOverview/ContentSecurityPolicy/Index.rst

```diff
@@ -46,7 +53,79 @@ need to be enabled.
 
 ..  versionchanged:: 13.0
 
-In TYPO3 backend the Content Security Policy is always enforced.
+    In TYPO3 backend the Content Security Policy is always enforced.
+
+Within the TYPO3 backend, a specific backend module is available to inspect policy
+violations / reports, and there is also a list to see all configured CSP rules,
+see section :ref:`content-security-policy-backend-rules`.
+
+
+..  _content-security-policy-terminology:
+
+Terminology
+=============
+
+This document will use very specific wording that is part of the CSP W3C specification,
+these terms are not "invented" by TYPO3. Since reading the W3C RFC can be very
+intimidating, here are a few key concepts:
+
+*   CSP consists of multiple rules (or "directives"), that are part of a "policy". This
+    policy says, what functionality the site's output is allowed to use.
+
+*   With that, several HTML tags can be controlled, like from which URLs images can be
+    requested from, if and from where iframes are allowed, if and from where JavaScripts
+    are allowed and so on. There is a long list of applicable directives, see
+    `https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy#directives`__
+    with specific identifiers like `default-src`, `img-src` and so on.
+
+*   Each directive may have several "attributes" (like the allowed URLs).
+
+*   Directives may build upon each other, a bit like CSS definitions (`Cascading Style Sheet`)
+    do. However, these are more meant to modify a basic rule on an earlier level,
+    and called "mutations". The relation of a "child rule" to it's "parent" is
+    also called "ancestor chain".
+
+*   A final policy is compiled of all these directives, and then sent as a HTTP
+    response header `Content-Security-Policy: ...`.
+
+*   In TYPO3, directives can be specified via PHP syntax (within Extensions) and
+    YAML syntax (within site configuration).
+
+*   These rules can influence each other, this is where the concept of "mutations" come in.
+    The "policy builder" of TYPO3 applies each configured mutation, no matter where it was
+    defined.
+
+*   Because of this, each mutation (directive definition) needs a specific "mode" that can
+    instruct, how this mutation is applied: Should an existing directive be
+    `appended`, `removed` or `added` to the final policy.
+
+*   Each directive is then applied in regard to its defined mode and can list one or more
+    "sources" with the values of additional parameters of a directive. Sources are
+    web site addresses / URLs.
+
+*   There are possible exemptions to directives for specific content created on specific
+    pages created by TYPO3 in your frontend (or backend modules). To verify, that these
+    exemptions are valid in a policy, a so-called "Nonce" (Hash) is created.
+
+    *   TYPO3 can manage these Nonces and apply them were configured.
+    *   Nonces are set from a pool of available hashes, and whenever a Nonce is used
```

Comment:
`\TYPO3\CMS\Core\Security\NoncePool` and `\TYPO3\CMS\Core\Security\Nonce` are not used in the scope of CSP - instead `\TYPO3\CMS\Core\Security\ContentSecurityPolicy\ConsumableNonce` is used. The `NoncePool` is used for anti-CSRF (see https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/12.0/Feature-97305-IntroduceCSRF-likeRequest-tokenHandling.html).

Thus, in CSP-context there is only one nonce for the whole request - that value is the same in all directives.

Reply:
Thanks, I mixed this up. Reworded the section.